docs/software/sciviz/paraview.md

```diff
@@ -0,0 +1,137 @@
+[](){#ref-uenv-paraview}
+# ParaView
+
+[ParaView](https://www.paraview.org/) is an open-source, multi-platform scientific data analysis and visualization tool that enables analysis and visualization of extremely large datasets. ParaView is both a general purpose, end-user application with a distributed architecture that can be seamlessly leveraged by your desktop or other remote parallel computing resources and an extensible framework with a collection of tools and libraries for various applications including scripting (using Python), web visualization (through trame and ParaViewWeb), or in situ analysis (with Catalyst).
+
+!!! note "uenvs"
+
+    [ParaView](https://www.paraview.org/) is provided on [ALPS][platforms-on-alps] via [uenv][ref-uenv].
+    Please have a look at the [uenv documentation][ref-uenv] for more information about uenvs and how to use them.
+
+
+## Running ParaView in batch mode with Python scripts
+
+The following sbatch script can be used as a template.
+
+=== "GH200"
+
+    ```bash
+    #SBATCH -N 1
+    #SBATCH --ntasks-per-node=4
+    #SBATCH --cpus-per-task=72
+    #SBATCH --gpus-per-task=1
+    #SBATCH -A <account>
+    #SBATCH --uenv=paraview/5.13.2:v2 --view=paraview
+    #SBATCH --hint=nomultithread
+
+    export MPICH_GPU_SUPPORT_ENABLED=0
+
+    srun --cpus-per-task=72 --cpu_bind=sockets /user-environment/ParaView-5.13/gpu_wrapper.sh /user-environment/ParaView-5.13/bin/pvbatch ParaViewPythonScript.py
+    ```
+    Current observation is that best performance is achieved using [one MPI rank per GPU][ref-slurm-gh200-single-rank-per-gpu]. How to run multiple ranks per GPU is described [here][ref-slurm-gh200-multi-rank-per-gpu].
+
+=== "Eiger"
+
+    ```bash
+    #SBATCH -N 1
+    #SBATCH --ntasks-per-node=144
+    #SBATCH -A <account>
+    #SBATCH --uenv=paraview/5.13.2:v2 --view=paraview
+    #SBATCH --hint=nomultithread
+
+    srun --cpus-per-task=144 /user-environment/ParaView-5.13/bin/pvbatch ParaViewPythonScript.py
+    ```
+
+
+## Using ParaView in client-server mode
+
+A ParaView server can connect to a remote ParaView client installed on your desktop. Make sure to use the same version on both sides. Your local ParaView GUI client needs to create a SLURM job with appropriate parameters. We recommend that you make a copy of the file `/user-environment/ParaView-5.13/rc-submit-pvserver.sh` to your $HOME, such that you can further fine-tune it.
+
+You will need to add the corresponding XML code to your local ParaView installation, such that the Connect menu entry recognizes the ALPS cluster. The following code would be added to your **local** `$HOME/.config/ParaView/servers.pvsc` file
+
+!!! Example "XML code to add to your local ParaView settings"
+```console
```

Comment:
This whole code block needs to be indented to be placed inside the example box (at the moment it is floating below an empty example box).

Furthermore, instead of `console` you can mark it as `xml` to get xml highlighting.